projects/fusion-e2e/tests/behavior/behavior-class/base-element.ts

```diff
@@ -0,0 +1,193 @@
+import {Locator, Page, test} from '@playwright/test';
```

Comment:
I have a few  notes regarding the consts file . (its not a part of the pr for some reason but it should be)

1. It appears that you don’t use most of the constants there within the alert component so remove them
2. If you do need some of them -  We need to decide if its right to expose the contestants  file to external users - maybe its something that should not be exported as a part of the library ? Not sure lets discuss it
3. Change the name to constants instead of consts :)

Reply:
1 and 3 Done
2 - lets discuss it